pandas/core/indexes/base.py

```diff
@@ -2450,10 +2450,24 @@ def _get_unique_index(self, dropna=False):
 
         Returns
         -------
-        loc : int if unique index, possibly slice or mask if not
+        loc : int if unique index, slice if monotonic index, else mask
+
+        Examples
+        ---------
+        >>> unique_index = pd.%(target_klass)s(list('abc'))
```

Comment:
hmm these are only valid for Index/CategoricalIndex. Do these examples show up for other indexes (IIR we are using this combined doc-string for all sub-classes). You'll have to do this as an additional ``@Appender`` to those classes only.

Reply:
I think I will do this all over again. Many of the ``.get_loc`` methods are subtlely different, so it's difficult to have a common doc-string.

Before I do that, could you please explain how ``IntervalIndex.get_loc`` gets its doc-string? From the code it seems it should have one, but it does....

Reply:
hmm, I think its inheriting it from Index.get_loc, instead this should use the appender method (guess it was omitted somehow).